notify/docs/sending-texts.md

```diff
@@ -49,6 +50,14 @@ For example:
     "amount_owing": "$15.50"
 }
 ```
+---
+
+`reference`
+
+**Optional**
+
+A unique identifier you create. This reference identifies a single unique notification or a batch of notifications. If you do not have a reference, you must pass in an empty string or null.
```

Comment:
"empty string" and "null" are language dependent.

I wonder if we just word this like "if you do not have a reference, you can omit this argument".

Reply:
How about I leave the last sentence out altogether? "Omit" to me means leaving it out completely which is not valid for the java client.

Reply:
Sounds good. The thing is marked as `Optional` anyway.